### PR TITLE
New feature - increment people properties

### PR DESCRIPTION
**Pull request by proteye**
Added new feature - increment people properties on both OS (ios, android).

### CODE REVIEW

**Comment by proteye** [comment]
@truenary
Hello! Please, check my PR and if it's ok then merge it.

**Comment by surenyonjan** [comment]
@proteye Will do it today

**Comment by proteye** [comment]
@surenyonjan Hello. Please, check my PR and merge it.

**Comment by surenyonjan** [comment]
> @surenyonjan Hello. Please, check my PR and merge it.

@proteye Will do it after few hours. Sorry was too busy in my own stuff. :|

**Comment by surenyonjan** [comment]
@proteye Just finished reviewing changes. Thanks for this new feature. I ran it in my iOS device but still need to run it in Android device (I only have iPhone device for now). Will run it in Android Device tomorrow first hour of my time. Meanwhile is it possible to remove those indentation fixes?

File: example/lib/main.dart [hunk 1]
```diff
@@ -5,11 +5,11 @@ import 'package:flutter/services.dart';
 import 'package:native_mixpanel/native_mixpanel.dart';
 
 void main() => runApp(MyApp(
-  mixpanel: Mixpanel(
-    shouldLogEvents: true,
-    isOptedOut: false,
-  ),
-));
+      mixpanel: Mixpanel(
```
Comment (surenyonjan) [inline example/lib/main.dart]: Please remove this indentation changes.

Comment (proteye) [reply example/lib/main.dart]: Thank you for review. I will do it. It made autoformat of Flutter.

**Comment by proteye** [comment]
@surenyonjan Hello. Please, update pub repository - https://pub.dev/packages/native_mixpanel to 0.1.2 version.

**Comment by sosuren** [comment]
Ahh sorry. Just pushed it.

**Pull request merged**